droidlet/tools/data_scripts/fetch_artifacts_from_aws.py

```diff
@@ -36,13 +36,16 @@ def fetch_artifact_from_aws(agent, artifact_name, model_name, checksum_file_name
     print("CHECKSUM: %r" % checksum_val)
 
     artifact_path = os.path.join(agent_path, artifact_name)
+    write_path = artifact_path
```

Comment:
Ah I know why this is happening. It is because of `--strip-components 1` used here
Now that models has one more directory : `models/nlu/` and `models/perception` compared to just `models` before we'll have to do `--strip-components 2` or write one directory outside (only for models).

Reply:
Thanks for moving this up. This is cleaner than the if else